### DISCUSSION TITLE
[Run issue]:Unable to Run Example（Nucleus.txt） from TOPAS-nBio due to Scoring Setup Error

### DISCUSSION THREAD

**Discussion by v587aaaaa3** [topas-nbio/TOPAS-nBio#50] [Run-Problem]
### Contact Details

weikaijie644@gmail.com

### What was the issue?

I tried to run the example (https://github.com/topas-nbio/TOPAS-nBio/tree/master/examples/geometry/nucleusModel/Nucleus.txt), but my computer prompted me with the following error: 

TOPAS is exiting due to a serious error in scoring setup.
 Sc/DNADamageScorer/FileVoxelCopyNumbersInsideNucleus file has to be specified.

I couldn't find a usable file in the supportFiles folder, and I hope to get some clarification.

### TOPAS-nBio category

Geometry

### Version

3.0

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by jschuemann** [comment]
Hi,
the path to the scorer is 
examples/scorers/DNAScorerStepByStep.txt  (from the TOPAS-nBio home).
I am not sure what happened there though, because that message is confusing, FileVoxelCopyNumbersInsideNucleus is not a variable as far as I know.
